<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>793</width>
    <height>399</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Demo3_9 QTreeWidget,QDockWidget</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QScrollArea" name="scrollArea">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>431</width>
      <height>301</height>
     </rect>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <widget class="QWidget" name="scrollAreaWidgetContents">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>429</width>
       <height>299</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="LabPicture">
        <property name="text">
         <string>待显示图片</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>793</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>目录树</string>
    </property>
    <addaction name="act_AddDirectory"/>
    <addaction name="act_AddFiles"/>
    <addaction name="act_DelNode"/>
    <addaction name="act_IterationgNodes"/>
    <addaction name="separator"/>
    <addaction name="act_Exit"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>视图</string>
    </property>
    <addaction name="act_Amplification"/>
    <addaction name="act_Zoomout"/>
    <addaction name="act_ActualSize"/>
    <addaction name="act_FitWidth"/>
    <addaction name="act_SuitableHeight"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="mainToolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="act_AddDirectory"/>
   <addaction name="act_AddFiles"/>
   <addaction name="act_DelNode"/>
   <addaction name="act_IterationgNodes"/>
   <addaction name="separator"/>
   <addaction name="act_Amplification"/>
   <addaction name="act_Zoomout"/>
   <addaction name="act_ActualSize"/>
   <addaction name="act_FitWidth"/>
   <addaction name="act_SuitableHeight"/>
   <addaction name="separator"/>
   <addaction name="act_FormFloating"/>
   <addaction name="act_FormVisible"/>
   <addaction name="separator"/>
   <addaction name="act_Exit"/>
  </widget>
  <widget class="QDockWidget" name="dockWidget">
   <property name="features">
    <set>QDockWidget::AllDockWidgetFeatures</set>
   </property>
   <property name="allowedAreas">
    <set>Qt::LeftDockWidgetArea|Qt::RightDockWidgetArea</set>
   </property>
   <property name="windowTitle">
    <string>图片目录树</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QTreeWidget" name="treeFiles">
       <column>
        <property name="text">
         <string>节点</string>
        </property>
        <property name="textAlignment">
         <set>AlignCenter</set>
        </property>
       </column>
       <column>
        <property name="text">
         <string>节点类型</string>
        </property>
       </column>
       <item>
        <property name="text">
         <string>图片文件</string>
        </property>
        <property name="icon">
         <iconset resource="../Resource/res.qrc">
          <normaloff>:/icons/icons/15.ico</normaloff>:/icons/icons/15.ico</iconset>
        </property>
        <property name="text">
         <string/>
        </property>
        <item>
         <property name="text">
          <string>分组节点</string>
         </property>
         <item>
          <property name="text">
           <string>图片节点</string>
          </property>
          <property name="icon">
           <iconset resource="../Resource/res.qrc">
            <normaloff>:/icons/icons/31.ico</normaloff>:/icons/icons/31.ico</iconset>
          </property>
          <property name="flags">
           <set>ItemIsSelectable|ItemIsUserCheckable|ItemIsEnabled</set>
          </property>
         </item>
        </item>
        <item>
         <property name="text">
          <string>分组2</string>
         </property>
         <item>
          <property name="text">
           <string>图片2</string>
          </property>
          <property name="icon">
           <iconset resource="../Resource/res.qrc">
            <normaloff>:/icons/icons/31.ico</normaloff>:/icons/icons/31.ico</iconset>
          </property>
          <property name="text">
           <string/>
          </property>
         </item>
        </item>
       </item>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <action name="act_AddDirectory">
   <property name="icon">
    <iconset resource="../Resource/res.qrc">
     <normaloff>:/icons/icons/open3.bmp</normaloff>:/icons/icons/open3.bmp</iconset>
   </property>
   <property name="text">
    <string>添加目录</string>
   </property>
   <property name="toolTip">
    <string>添加目录</string>
   </property>
  </action>
  <action name="act_AddFiles">
   <property name="icon">
    <iconset resource="../Resource/res.qrc">
     <normaloff>:/icons/icons/824.bmp</normaloff>:/icons/icons/824.bmp</iconset>
   </property>
   <property name="text">
    <string>添加文件</string>
   </property>
   <property name="toolTip">
    <string>添加文件</string>
   </property>
  </action>
  <action name="act_DelNode">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="../Resource/res.qrc">
     <normaloff>:/icons/icons/delete1.bmp</normaloff>:/icons/icons/delete1.bmp</iconset>
   </property>
   <property name="text">
    <string>删除节点</string>
   </property>
   <property name="toolTip">
    <string>删除节点</string>
   </property>
  </action>
  <action name="act_IterationgNodes">
   <property name="icon">
    <iconset resource="../Resource/res.qrc">
     <normaloff>:/icons/icons/fold.bmp</normaloff>:/icons/icons/fold.bmp</iconset>
   </property>
   <property name="text">
    <string>遍历节点</string>
   </property>
   <property name="toolTip">
    <string>遍历节点</string>
   </property>
  </action>
  <action name="act_Exit">
   <property name="icon">
    <iconset resource="../Resource/res.qrc">
     <normaloff>:/icons/icons/exit.bmp</normaloff>:/icons/icons/exit.bmp</iconset>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
   <property name="toolTip">
    <string>退出</string>
   </property>
  </action>
  <action name="act_Amplification">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="../Resource/res.qrc">
     <normaloff>:/icons/icons/418.bmp</normaloff>:/icons/icons/418.bmp</iconset>
   </property>
   <property name="text">
    <string>放大</string>
   </property>
   <property name="toolTip">
    <string>放大</string>
   </property>
  </action>
  <action name="act_Zoomout">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="../Resource/res.qrc">
     <normaloff>:/icons/icons/416.bmp</normaloff>:/icons/icons/416.bmp</iconset>
   </property>
   <property name="text">
    <string>缩小</string>
   </property>
   <property name="toolTip">
    <string>缩小</string>
   </property>
  </action>
  <action name="act_ActualSize">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="../Resource/res.qrc">
     <normaloff>:/icons/icons/414.bmp</normaloff>:/icons/icons/414.bmp</iconset>
   </property>
   <property name="text">
    <string>实际大小</string>
   </property>
  </action>
  <action name="act_FitWidth">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="../Resource/res.qrc">
     <normaloff>:/icons/icons/424.bmp</normaloff>:/icons/icons/424.bmp</iconset>
   </property>
   <property name="text">
    <string>适合宽度</string>
   </property>
  </action>
  <action name="act_SuitableHeight">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="../Resource/res.qrc">
     <normaloff>:/icons/icons/426.bmp</normaloff>:/icons/icons/426.bmp</iconset>
   </property>
   <property name="text">
    <string>适合高度</string>
   </property>
  </action>
  <action name="act_FormFloating">
   <property name="icon">
    <iconset resource="../Resource/res.qrc">
     <normaloff>:/icons/icons/814.bmp</normaloff>:/icons/icons/814.bmp</iconset>
   </property>
   <property name="text">
    <string>窗体浮动</string>
   </property>
  </action>
  <action name="act_FormVisible">
   <property name="icon">
    <iconset resource="../Resource/res.qrc">
     <normaloff>:/icons/icons/414.bmp</normaloff>:/icons/icons/414.bmp</iconset>
   </property>
   <property name="text">
    <string>窗体可见</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../Resource/res.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>act_Exit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>396</x>
     <y>199</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
